### DISCUSSION TITLE
Runelite Plugin Idea

### DISCUSSION THREAD

**Discussion by KitaSenpai** [runelite/runelite#17718] [Ideas]
So I have an "idea" for a plugin.  It would be specifically geared towards those doing farming/herb runs.  My idea goes a little bit like this.  Say you're doing a Herb Run, or farming in general.  (For this example we'll do an Herb Run) The plugin would do the following.

1. Tells you where your herbs are located.
2. Tells you if the herb requires a cure/is dead
3. Shows you what tier your plant currently is
4. Shows you if your plant has no fertilizer/normal/super
5. Shows you if your plant is ready to be harvested.

I'm not entirely sure how "plausible" this idea is, or how "feasible" it would be in game.  But i'd love some ideas on if this was to be able to come to fruition how it could potentially be made better/changed to be beneficial to others.

**Comment by DrJam** [comment]
All 5 of these are covered by the herb section of the Time Tracking plugin:
![image](https://github.com/runelite/runelite/assets/3051130/1d364420-dfac-435b-be3b-0c1379385e4e)

**Reply by KitaSenpai** [reply]
funny enough...  I actually never noticed this...  and I feel like i've looked for this...

**Reply by DrJam** [reply]
Make sure the plugin is enabled and then look for the clock icon in the sidebar
![image](https://github.com/runelite/runelite/assets/3051130/80c30261-e2c8-4191-99b2-802ff09f0921)
